packages/@okta/vuepress-site/docs/guides/oin-oidc-guide/protocol-level-requirements/index.md

```diff
@@ -0,0 +1,91 @@
+  * `state` &mdash; value to be returned in the token. The client application can use it to remember the state of its interaction with the end user at the time of the authentication call.
+* Your application sends a redirect response to the browser containing the composed Authentication Request. The browser interprets the redirect, and sends the Authentication Request to the `/authorize` endpoint on the Okta Authorization Server over secure HTTPs.
+* Okta then either sends the browser to an Okta sign-in page which asks the End User to provide authentication and consent to the requested scopes, or confirms that the user is already authenticated
+* The End User provides proof of identity (using any of the supported authentication methods) and gives consent for any requested claims defined in your `scopes` parameter
+* The browser receives an authorization code response from the Okta Authorization Server. If there is an error or the authentication fails for any reason, the error returned to the End User through the browser. If the authentication is successful, the Authorization Server issues an authorization `code` value and includes it in a redirect that sends the browser over to the Client at the Login Redirect URI callback location.
+* The Client (your application) then submits a request to the Okta /token endpoint using the authorization code and the Client Secret. If the code and secret are both valid, then Okta returns:
+  * `access_token` &mdash; this token is restricted to give access only to what the End User authorized for your application
+  * `token_type=Bearer` &mdash; this identifies the authentication method used
+  * `expires_in` &mdash; the expiration time for the access token (in seconds)
+  * `scope` &mdash; the scopes that are contained in the access token
+  * `id_token` &mdash; the ID token contains the specific info (in a JWT format) about your user that is needed by the application
+  * Optional. `refresh_token` &mdash; this token is returned if the `offline_access` scope is granted
+* The access token is then used by your application to retrieve the protected resources and information requested at the Resource Server for the End User.
+
+The Authorization code process flow depends on Okta and your SaaS back-end systems being able to securely communicate the Client Secret.
+
+Note that your application needs to verify user identity through a user store, otherwise your app could be spoofed by fake applications in the Android or Apple app stores or an End User could view and possibly modify your code. For this reason, OIDC native, mobile, and single page application (SPA) app integrations are not acceptable for OIDC app integrations in the OIN. Your application code must be set up in a flow so that your client application talks to your SaaS back-end which then communicates with Okta.
```

Comment:
```suggestion
Note that your application needs to verify user identity through a user store, otherwise your app could be spoofed by fake applications in the Android or Apple app stores or an End User could view and possibly modify your code. For this reason, OIDC native, mobile, and single page application (SPA) app integrations are not acceptable for OIDC app integrations in the OIN. Your application code must be set up in a flow so that your client application talks to your SaaS back-end which then communicates with Okta.
```

What does "OIDC native" mean?

Reply:
Yeah, I can see how that's confusing. I can remove it there.